src/host/CopyFromCharPopup.cpp

```diff
@@ -15,7 +15,7 @@ CopyFromCharPopup::CopyFromCharPopup(SCREEN_INFORMATION& screenInfo) :
 }
 
 // Routine Description:
-// - This routine handles the delete from cursor to char char popup.  It returns when we're out of input or the user has entered a char.
+// - This routine handles the delete from cursor to char popup.  It returns when we're out of input or the user has entered a char.

```

Comment:
Of the doubled words, this is the one about which I'm the least certain.

Reply:
this looks fine to me